trace-dispatcher/doc/trace-dispatcher.md

```diff
@@ -555,18 +650,49 @@ Because all these tracers are defined as part of the __dispatcher__ definition,
 
 3. This __dispatcher__ (in other words, the set of __tracers__ it is composed of) is given as an argument to the rest of the program, which then distributes them across its components and then begins execution.
 
+## Cardano tracer
+
+We provide a standard interface to construct a tracer to be used within cardano node.
+The tracer gets as arguments the backends: 'trStdout', 'trForward' and 'mbTrEkg'.
+The tracer gets as argument a 'name', which is appended to its namespace. The tracer gets as
+arguments 'namesFor', 'severityFor' and 'privacyFor' functions, to set the logging context accordingly. The returned tracer need to be configured with a configuration for the specification of filtering, detailLevel, frequencyLimiting and backends with a configuration before use.
```

Comment:
The camel casing in the above sentence and the single quoted words earlier  suggests these are identifiers?  If so, please backtick rather than leave them bare of single quoted.

Reply:
Done